adcc/modified_transition_moments.py

```diff
@@ -72,10 +74,43 @@ def compute_modified_transition_moments(gs_or_matrix, dipole_operator,
                         "libadcc.OneParticleOperator.")
 
     if method.is_core_valence_separated:
-        raise NotImplementedError("compute_modified_transition_moments "
-                                  "not implemented for CVS")
-
+        if method.name == "cvs-adc1":
+            return AmplitudeVector(dipole_operator['o2v1'])
```

Comment:
That should also be a separate function further below. It feels strange to have this return early as a special case.